benchmark/runner/mini_runners.cpp

```diff
@@ -0,0 +1,1161 @@
+  static constexpr execution::query_id_t OP_DECIMAL_DIVIDE_QID = execution::query_id_t(7);
+  static constexpr execution::query_id_t OP_DECIMAL_GEQ_QID = execution::query_id_t(8);
+
+  static constexpr execution::query_id_t SEQ_SCAN_COL_1_QID = execution::query_id_t(9);
+  static constexpr execution::query_id_t SEQ_SCAN_COL_4_QID = execution::query_id_t(10);
+  static constexpr execution::query_id_t SEQ_SCAN_COL_8_QID = execution::query_id_t(11);
+  static constexpr execution::query_id_t SEQ_SCAN_COL_15_QID = execution::query_id_t(12);
+
+  static constexpr execution::query_id_t BULK_INS_COL_1_QID = execution::query_id_t(13);
+  static constexpr execution::query_id_t BULK_INS_COL_4_QID = execution::query_id_t(14);
+  static constexpr execution::query_id_t BULK_INS_COL_8_QID = execution::query_id_t(15);
+  static constexpr execution::query_id_t BULK_INS_COL_15_QID = execution::query_id_t(16);
+
+  static constexpr execution::query_id_t SORT_COL_1_QID = execution::query_id_t(17);
+  static constexpr execution::query_id_t SORT_COL_4_QID = execution::query_id_t(18);
+  static constexpr execution::query_id_t SORT_COL_8_QID = execution::query_id_t(19);
+  static constexpr execution::query_id_t SORT_COL_15_QID = execution::query_id_t(20);
+
+  static constexpr execution::query_id_t UPDATE_COL_1_QID = execution::query_id_t(21);
+  static constexpr execution::query_id_t UPDATE_COL_4_QID = execution::query_id_t(22);
+  static constexpr execution::query_id_t UPDATE_COL_8_QID = execution::query_id_t(23);
+  static constexpr execution::query_id_t UPDATE_COL_15_QID = execution::query_id_t(24);
+
+  static constexpr execution::query_id_t AGG_COL_1_QID = execution::query_id_t(25);
+  static constexpr execution::query_id_t AGG_COL_4_QID = execution::query_id_t(26);
+  static constexpr execution::query_id_t AGG_COL_8_QID = execution::query_id_t(27);
+  static constexpr execution::query_id_t AGG_COL_15_QID = execution::query_id_t(28);
+
+  static constexpr execution::query_id_t HJ_COL_1_QID = execution::query_id_t(29);
+  static constexpr execution::query_id_t HJ_COL_4_QID = execution::query_id_t(30);
+  static constexpr execution::query_id_t HJ_COL_8_QID = execution::query_id_t(31);
+  static constexpr execution::query_id_t HJ_COL_15_QID = execution::query_id_t(32);
+
+  static constexpr execution::query_id_t NLJ_COL_1_QID = execution::query_id_t(33);
+  static constexpr execution::query_id_t NLJ_COL_4_QID = execution::query_id_t(34);
+  static constexpr execution::query_id_t NLJ_COL_8_QID = execution::query_id_t(35);
+  static constexpr execution::query_id_t NLJ_COL_15_QID = execution::query_id_t(36);
+
+  static constexpr execution::query_id_t IDX_KEY_1_QID = execution::query_id_t(37);
+  static constexpr execution::query_id_t IDX_KEY_4_QID = execution::query_id_t(38);
+  static constexpr execution::query_id_t IDX_KEY_8_QID = execution::query_id_t(39);
+  static constexpr execution::query_id_t IDX_KEY_15_QID = execution::query_id_t(40);
```

Comment:
I don't think we really want to do this. I think we can just use a simple counter and increment with every query we run. Otherwise we have to fix these numbers every time want to add/modify the number of columns we're going to test for a operating unit.

Reply:
Done.